packages/framework/src/schemas/providers/chat/index.ts

```diff
@@ -1,5 +1,15 @@
+import { genericProviderSchemas } from '../generic';
 import { slackProviderSchemas } from './slack';
 
 export const chatProviderSchemas = {
   slack: slackProviderSchemas,
+  discord: genericProviderSchemas,
+  getstream: genericProviderSchemas,
+  grafanaOnCall: genericProviderSchemas,
+  mattermost: genericProviderSchemas,
+  msTeams: genericProviderSchemas,
+  rocketChat: genericProviderSchemas,
+  ryver: genericProviderSchemas,
+  whatsappBusiness: genericProviderSchemas,
+  zulip: genericProviderSchemas,
```

Comment:
Let's import and use the ProviderId enums from here https://github.com/novuhq/novu/blob/vitest-test/libs/shared/src/consts/providers/provider.enum.ts to ensure that we always add the generic schema here and don't regress the provider experience when new providers are added